readme.md

```diff
@@ -56,6 +41,63 @@ Import certificate into truststore:
         -file doej.cer \
         -keystore doej.truststore
 
+## Gradle
+
+If you are already familiar with gradle, or have it installed on your system feel free to skip ahead to the next section in the documentation.
+
+### The Gradle Wrapper
+
+Once you have cloned the skeleton, generated keys, and changed into its directory, you can build and run it by typing the following into the command line
+
+    $ ./gradlew build
+    $ ./gradlew run
+
+While this works fine, you can save yourself the './' and the 'w' by installing gradle and adding it to your path.
+
+### Installing Gradle
```

Comment:
these instructions are correct, but `gradlew` actually downloads Gradle automatically ("When you start a Gradle build via the wrapper, Gradle will be automatically downloaded and used to run the build. "):
https://docs.gradle.org/current/userguide/gradle_wrapper.html

i believe it installs the binary somewhere under `~/.gradle`